logging.go

```diff
@@ -259,55 +259,15 @@ func (wdest writerDestructor) Destruct() error {
 	return wdest.Close()
 }
 
-// StandardLibLog configures the default Go standard library
+// BaseLog configures the default Go standard library
 // global logger in the log package. This is necessary because
 // module dependencies which are not built specifically for
 // Caddy will use the standard logger. This is also known as
 // the "sink" logger.
```

Comment:
This comment should be moved to `SinkLog` I think, and this should be changed to properly explain what `BaseLog` does.